app/assets/stylesheets/projects.css.scss

```diff
@@ -27,3 +27,21 @@ h5 span.favorite {
 .project_links {
   margin-bottom: 6%;
 }
+
+.tag_list {
+  border-radius: 5px;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.